packages/cairo.rb

```diff
@@ -10,9 +10,11 @@ class Cairo < Package
   depends_on 'fontconfig'
   depends_on 'libpng'
   depends_on 'pixman'
+  depends_on 'freetype'   # pango requires cairo with freetype
```

Comment:
If cairo acesses only fontconfig, I agree with you.  But it accesses both freetype and fontconfig.  Cairo generates multiple functions depends on existence of other packages, so leave them both increase the safeness against to the future modifications.